target/linux/ath79/nand/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -12,6 +12,9 @@ case "$FIRMWARE" in
 	8dev,rambutan)
 		caldata_extract "caldata" 0x1000 0x800
 		;;
+	linksys,ea4500-v3)
+		caldata_extract "caldata" 0x1000 0x440
+		;;
```

Comment:
Try to convert calibration data to nvmem, please refer https://github.com/openwrt/openwrt/commit/48625a04453b4f5775db2d2a5abebf977b81344c.

Reply:
Done.